fvm/src/engine/mod.rs

```diff
@@ -572,6 +572,7 @@ impl Engine {
             last_memory_bytes: memory_bytes,
             last_charge_time: GasTimer::start(),
             memory: self.inner.dummy_memory,
+            wasm_prices: self.inner.config.wasm_prices,
```

Comment:
In the future, we may want to store a copy of the entire engine here. But that's not necessary yet.

Reply:
Honestly, no reason. I'm pulling out a small piece of the engine config here so I considered just stashing the engine. But yeah... this comment makes no sense without knowing my local history of this patch.